py/runtime.h

```diff
@@ -33,6 +33,7 @@
 // For use with mp_call_function_1_from_nlr_jump_callback.
 #define MP_DEFINE_NLR_JUMP_CALLBACK_FUNCTION_1(ctx, f, a) \
     nlr_jump_callback_node_call_function_1_t ctx = { \
+        .callback = {.prev = NULL, .fun = NULL}, \
```

Comment:
I'm not quite sure about this change... this macro is always followed by a call to `nlr_push_jump_callback()` which is responsible for initialising this sub-struct.  So it's a bit of a waste to set it to NULL here.

Maybe a better way is to change this macro to:
```py
nlr_jump_callback_node_call_function_1_t ctx;
ctx.func = f;
ctx.arg = a;
```
or something similar?

Reply:
Ok.